### ISSUE TITLE
Module 3 - Nb 4 - Failed ingestion

### ISSUE THREAD

**Issue by psantus** [aws-samples/amazon-sagemaker-feature-store-end-to-end-workshop#67]
Before
> %%time
customers_fg.ingest(data_frame=customers_df, max_processes=16, wait=True)
logger.info(f'{len(customers_df)} customer records ingested into feature group: {customers_feature_group_name}')

You should include
> customers_df = customers_df.convert_dtypes(infer_objects=True, convert_boolean=False)

Otherwise feature.ingest fails on age categories.

**Comment by pmhargis-aws** [comment]
Yes, I added `convert_dtypes()` to other notebooks, and will add it here too.
